src/olympia/editors/models.py

```diff
@@ -437,10 +437,10 @@ def get_event(cls, addon, status, version=None, post_review=False,
                 if version is None:
                     raise AutoApprovalSummary.DoesNotExist
                 weight = version.autoapprovalsummary.weight
-            except AutoApprovalSummary.DoesNotExist as exp:
+            except AutoApprovalSummary.DoesNotExist as exception:
```

Comment:
hmm, I actually meant naming it `exc` instead of `exp` but `exception` works too